### ISSUE TITLE
Confirmation Email did not provide any google meet Link

### ISSUE THREAD

**Issue by lawweiliang** [calcom/cal.com#5310]
Found a bug? Please fill out the sections below. 👍

### Issue Summary

System did not automatically generate an google meet link. 

![image](https://user-images.githubusercontent.com/34936929/199166907-158f18b1-295e-42b5-a2f2-b20eebb2930e.png)

### Steps to Reproduce

1. After user booked an event.
2. User will received an confirmation email, inside that email, there is no location meeting link.

**Comment by milospuac** [comment]
Hi @lawweiliang Do you have a google calendar connected to your cal.com account? 

what is your cal.com username? I'd like to investigate this

**Comment by lawweiliang** [comment]
@milospuac, my username is lawweiliang.

Thanks for taking your time to investigate this issue. Have a lovely day. 😄

**Comment by lawweiliang** [comment]
@milospuac 

Let't get back one step and think a little bit. Why I need to connect google calendar first then only can use google meet. 
It does not make sense right? What do you think?

You try to create an account over calendly and play around with that. Then, you will get some idea.

**Comment by PeerRich** [comment]
its a limitation by google. they only let you generate a google meet link if you have a google calendar connected

**Comment by lawweiliang** [comment]
> . they only let you generate a google meet link if yo

Today only I know about this, thanks for your clarification @PeerRich.

**Comment by michaelaflores** [comment]
Just to avoid an obvious footgun here, I would consider disabling the option to set Google Meet as the event type when Google Calendar is not the destination calendar. (I thought perhaps at least connecting my Google account would solve this, but no such luck, it really only allows the link when creating a Google Calendar event).

**Comment by michaelaflores** [comment]
@PeerRich I wonder if there's a solution here where a user who has Google Meet as their video conferencing but a non-Google calendar as their default could have a shadow event created in GCal for the purpose of getting the Meet link, but still have the event added to their preferred calendar? Is this something where a user could set up a workflow for it? (new to cal.com so may be way off that workflows could do something like this)

**Comment by PeerRich** [comment]
there is a new solution in the making where google meet is optional

**Comment by PeerRich** [comment]
cc @joeauyeung

**Comment by PeerRich** [comment]
TLDR: google meet requries an active google calendar
